### PR TITLE
add RC/release build configure parameters to release_process.md

### PR DESCRIPTION
**Pull request by sxa555**
FYA @pshipton @andrew-m-leonard

### CODE REVIEW

File: doc/processes/release_process.md [hunk 1]
```diff
@@ -109,7 +109,9 @@ them.  They are merely points in time.
 Update the Extensions branches to pull the tagged levels from the `openj9` 
 & `openj9-omr` release branches.
 1. Rebuild the tagged levels and ensure they pass the quality bar defined by 
-AdoptOpenJDK.
+AdoptOpenJDK. These builds need to have `--with-milestone=fcs` (JDK8) or
```
Comment (pshipton) [inline doc/processes/release_process.md]: Please explain why these options are required and what they do.

**Comment by pshipton** [comment]
@sxa555 please note you need to sign the Eclipe ECA (see [here](https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md))and sign your commit in order for the ip-validation check to pass so we can accept your contribution.

File: doc/processes/release_process.md (hunk 1 above)
Comment (andrew-m-leonard) [reply doc/processes/release_process.md]: These configure options are what openjdk's makefiles use to set variables that determine what appears in the java -version output, as discussed a while back.. The default ones will mean you have "user" build info in the strings, username's etc...

Comment (pshipton) [reply doc/processes/release_process.md]: Thanks Andrew. To be clear, what I meant is that the document should be updated with that information.

**Comment by sxa555** [comment]
Signed and hopefully added enough information - if you need any more let me know @pshipton

File: doc/processes/release_process.md (hunk 1 above)
Comment (sxa555) [reply doc/processes/release_process.md]: I've added a bit more info to address this - let me know if it's adequate or if more detail is required.

**Pull request merged**